dnf/base.py

```diff
@@ -1442,7 +1442,7 @@ def trans_upgrade(query, remove_query, comps_pkg):
         def trans_install(query, remove_query, comps_pkg):
             if self.conf.multilib_policy == "all":
                 if not comps_pkg.requires:
-                    self._install_multiarch(query, strict=False)
+                    self._install_multiarch(query, strict=True)
```

Comment:
In this case strict here should be an inherited variable from group_install function.

Reply:
yes, I figured that's what was going on there, but the association was lost in a refactor a while back, and isn't super straightforward to restore, as these are distinct operations - we'd have to find somewhere appropriate to stash the `strict` setting between the `group_install` and the `resolve`. Really, this whole `strict` mechanism feels somewhat wobbly to me - it's never clear whether `strict` is supposed to be a property of the overall DNF configuration, or this specific `Base` instance's configuration, or a transaction, or some specific operation that feeds into a transaction...the contexts seem to get mixed up a lot. Unless I'm missing something.